src/cjs-transform.js

```diff
@@ -93,14 +96,18 @@ class CJSTransform extends Plugin {
     const resolve = require('rollup-plugin-node-resolve');
     const commonjs = require('rollup-plugin-commonjs');
 
+    let fullPath = path.posix.join(this.projectRoot, relativePath);
+    if (relativePath.startsWith(NODE_MODULES)) {
+      fullPath = resolveSync(relativePath.slice(NODE_MODULES.length), {
+        basedir: this.projectRoot,
+      });
+    }
```

Comment:
Can you add an `else` here that throws an error? I don't think we ever expect for `relativePath` to **not** include `node_modules/` do we?

Reply:
good thinking! I switched it to throw early if we don't detect the "node_modules/" prefix.